src/composables/accountSelector.ts

```diff
@@ -0,0 +1,194 @@
+import {
+  computed, ref, toRef, watch,
+} from 'vue';
+
+import type {
+  IAccountSelectorEntry,
+  ITransaction,
+} from '@/types';
+import {
+  ACCOUNT_SELECT_TYPE_FILTER,
+  AccountSelectTypeFilter,
+  PROTOCOLS,
+  TX_DIRECTION,
+} from '@/constants';
+import { createCustomScopedComposable, getDefaultAccountLabel } from '@/utils';
+import {
+  useAccounts, useAddressBook, useLatestTransactionList, useTransactionData,
```

Comment:
We have this unwritten rule that if we are splitting the line we are putting each item in separate line.

Reply:
Updated